cypress/support/fragments/bulk-edit/bulk-edit-actions.js

```diff
@@ -128,5 +128,28 @@ export default {
 
   newBulkEdit() {
     cy.do(Button('New bulk edit').click());
+  },
+
+  verifyUsersActionDropdownItems() {
+    cy.expect([
+      DropdownMenu().find(Checkbox({ name: 'active', checked: true })).exists(),
+      DropdownMenu().find(Checkbox({ name: 'lastName', checked: true })).exists(),
+      DropdownMenu().find(Checkbox({ name: 'firstName', checked: true })).exists(),
+      DropdownMenu().find(Checkbox({ name: 'barcode', checked: true })).exists(),
+      DropdownMenu().find(Checkbox({ name: 'patronGroup', checked: true })).exists(),
+      DropdownMenu().find(Checkbox({ name: 'username', checked: true })).exists(),
+      DropdownMenu().find(Checkbox({ name: 'email', checked: false })).exists(),
+      DropdownMenu().find(Checkbox({ name: 'expirationDate', checked: false })).exists(),
+    ]);
+  },
+
+  verifyCheckedDropdownMenuItem() {
+    cy.do(DropdownMenu().find(Checkbox({ name: 'firstName'})).click());
```

Comment:
Move checkbox to const

Reply:
here is makes sense to move  DropdownMenu() to const

Reply:
done